test/t/unit/test_unit_load.py

```diff
@@ -1,12 +1,27 @@
 import pytest
+import os
 
-from conftest import assert_bash_exec, bash_env_saved
+from conftest import assert_bash_exec, bash_env_saved, prepare_fixture_dir
 
 
 @pytest.mark.bashcomp(cmd=None, cwd="_comp_load")
 class TestCompLoad:
-    def test_userdir_1(self, bash):
+    @pytest.fixture
+    def fixture_dir(self, request, bash):
```

Comment:
Let's add a short docstring/comment here as well why we're constructing the dir manually this way.

Pre-approved/good to merge with that change.

Reply:
Thank you for the comment! I added a description in the docstring.